### ISSUE TITLE
Windows ten talon task meu icon appears to be invisible to nvda screen reader

### ISSUE THREAD

**Issue by tararoys** [talonvoice/talon#535]
When accessing the talon icon on the lower left hand corner on the task bar of a windows ten operating system, using this screen reader software NVDA, highlighting the talent icon with the system focus  does not read out talon's name.  from the perspective of the NVDA screen meter software,  the talent icon is unnamed and unknown. 

 to reproduce this bug, on a windows ten operating system
   1. Start talon 
   2.  drag the talon icon from the windows ten notification overflow center to the task bar 
   3.  Start NVDA space screen reader software 
   4.  Access the task bar with the keyboard shortcut windows - t   ( talent command super trap)
   5.  access the notification center  by pressing the tab key (probably) Twice until you get to the thing that says notification  chevron button ( talon command tab)
   6.  Sentence go through items in the notification center by using the right arrow key (  talon command go right)
   7.  notice that when the talon icon is focused with a square around it,  NVDA says nothing about it.

Video of error:  

https://youtube.com/shorts/7sdT50mEPLs?feature=share

**Comment by lunixbochs** [comment]
Can you retest this on the latest talon-rust? I think it was a Qt bug, I wrote my own tray menu code for talon-rust.

**Comment by lunixbochs** [comment]
I believe this is fixed. please reopen or open a new issue if you still encounter it.
